src/dashboard/src/components/administration/forms.py

```diff
@@ -60,6 +60,17 @@ def save(self, *args, **kwargs):
 
 
 class StorageSettingsForm(SettingsForm):
+
+    class StripCharField(forms.CharField):
```

Comment:
I think using the `clean_storage_service_apikey` would be better here. https://docs.djangoproject.com/en/1.8/ref/forms/validation/#cleaning-a-specific-field-attribute